grpc.go

```diff
@@ -388,32 +419,22 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
-	libContContainer, err := factory.Create(req.ContainerId, config)
+	ctr.container, err = factory.Create(req.ContainerId, config)
 	if err != nil {
 		return emptyResp, err
 	}
+	ctr.config = *config
 
-	builtProcess, err := buildProcess(req.OCI.Process, req.ExecId)
+	ctr.initProcess, err = buildProcess(req.OCI.Process, req.ExecId)
 	if err != nil {
 		return emptyResp, err
 	}
 
-	container := &container{
-		id:          req.ContainerId,
-		initProcess: builtProcess,
-		container:   libContContainer,
-		config:      *config,
-		processes:   make(map[string]*process),
-		mounts:      mountList,
-	}
-
-	a.sandbox.setContainer(req.ContainerId, container)
-
-	if err := a.execProcess(container, container.initProcess, true); err != nil {
+	if err = a.execProcess(ctr, ctr.initProcess, true); err != nil {
 		return emptyResp, err
 	}
 
-	return emptyResp, a.postExecProcess(container, container.initProcess)
```

Comment:
doing this little rename cleanup in a separate commit would have made review a touch easier ;-)